core/src/test/scala/unit/kafka/message/MessageCompressionTest.scala

```diff
@@ -29,24 +26,6 @@ import org.junit.Assert._
 class MessageCompressionTest extends JUnitSuite {
 
   @Test
-  def testLZ4FramingV0() {
-    val output = CompressionFactory(LZ4CompressionCodec, Message.MagicValue_V0, new ByteArrayOutputStream())
-    assertTrue(output.asInstanceOf[KafkaLZ4BlockOutputStream].useBrokenFlagDescriptorChecksum())
-
-    val input = CompressionFactory(LZ4CompressionCodec, Message.MagicValue_V0, new ByteArrayInputStream(Array[Byte](0x04, 0x22, 0x4D, 0x18, 0x60, 0x40, 0x1A)))
-    assertTrue(input.asInstanceOf[KafkaLZ4BlockInputStream].ignoreFlagDescriptorChecksum())
-  }
-
-  @Test
-  def testLZ4FramingV1() {
-    val output = CompressionFactory(LZ4CompressionCodec, Message.MagicValue_V1, new ByteArrayOutputStream())
-    assertFalse(output.asInstanceOf[KafkaLZ4BlockOutputStream].useBrokenFlagDescriptorChecksum())
-
-    val input = CompressionFactory(LZ4CompressionCodec, Message.MagicValue_V1, new ByteArrayInputStream(Array[Byte](0x04, 0x22, 0x4D, 0x18, 0x60, 0x40, -126)))
-    assertFalse(input.asInstanceOf[KafkaLZ4BlockInputStream].ignoreFlagDescriptorChecksum())
-  }
```

Comment:
I think these tests were meant to check that we create the input/output stream with the right parameters. Maybe we should add something similar to `CompressionTypeTest` since the logic for doing that is now in that class?